### ISSUE TITLE
Lavaplayer stopped playing youtube links

### ISSUE THREAD

**Issue by Arskamann** [sedmelluq/lavaplayer#724]
So yeah about 1,5 weeks ago I noticed that my bot suddenly does not play youtube links. Local files work flawlessly but when loading a track from youtube, the player just spits it out instantly. Bot has been working without problems for 2 years at least.

No exceptions are thrown. No errors. Nothing...

**Comment by Pthread-Create** [comment]
same for by bot :(
only LOAD_FAILED

**Comment by milosz08** [comment]
Yeah, same issue. Surprisingly, soundcloud works correctly. But i found the sollution. Change this dep to this fork: https://github.com/lavalink-devs/lavaplayer (latest 2.0.1 version works fine for me).
